src/components/data/FileBrowser.js

```diff
@@ -248,6 +249,13 @@ const BucketBrowserTable = ({
                             },
                             [label]
                           ),
+                          h(ClipboardButton, {
+                            'aria-label': `Copy ${object.name} to clipboard`,
+                            className: 'cell-hover-only',
+                            style: { marginLeft: '1rem' },
```

Comment:
`1rem` is equal to the font size. 1rem usually ends up being a little much for horizontal spacing. Suggest using `1ch` here to add a character's _width_ as spacing.

```suggestion
                            style: { marginLeft: '1ch' },
```

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths